syntax = "proto3";

package client;

// The insert client service definition.
service CrudClientService {
  // insert new client
  rpc insert(GrpcClient) returns (GrpcClient) {}
  // update  client
  rpc update(GrpcClient) returns (GrpcClient) {}
  // search client by ID

  rpc search_by_id(GrpcClient) returns (GrpcClient) {}
  // delete Client by ID
  rpc delete_by_id(GrpcClient) returns (GrpcClient) {}
}

message GrpcClient{
  string name = 1;
  string cpf = 2;
  string id = 3;
  string favorite_food = 4;
}